### PR TITLE
[DSY-2278/2287/2290] - Theme customization - Badge | New attributes at StandardAppBarTop 

### PR DESCRIPTION
**Pull request by gsleonel**
# Description

Update the badge theme attribute values (color, typography and borderradius) according to each brand.
Adds height, width, and menu attributes at StandarAppBarTop.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit tests (`BadgeTest.kt`)
- [x] Screenshot test (`BadgeActivityScreenshotTest.kt`)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: designsystem/src/main/kotlin/com/natura/android/badge/Badge.kt [hunk 1]
```diff
@@ -42,6 +42,7 @@ class Badge @JvmOverloads constructor(
     private var badgeAttributeArray: TypedArray
     private var attrNumber: Int = 0
     private var attrVisibility: Boolean = true
+    private var isFontWeight: Boolean = false
```
Comment (yasminnvaz) [inline designsystem/src/main/kotlin/com/natura/android/badge/Badge.kt:45]: por que temos um booleano para font weight?

File: designsystem/src/main/kotlin/com/natura/android/badge/BadgeDrawable.kt [hunk 2]
```diff
@@ -39,40 +41,35 @@ class BadgeDrawable(
...
-            getDimenFromTheme(R.attr.sizeTiny) / 2,
-            getDimenFromTheme(R.attr.sizeTiny) / 2,
-            circlePaint
-        )
-    }
-
-    private fun createPulseVariant(canvas: Canvas) {
-        initializePaint()
+        val bounds = bounds
+        val badgeWith = getDimenFromTheme(R.attr.sizeTiny).toInt()
```
Comment (yasminnvaz) [inline designsystem/src/main/kotlin/com/natura/android/badge/BadgeDrawable.kt:61]: acho que temos um typo aqui. Não seria `badgeWidth`? 

outra dúvida: a altura do badge vem do contrato dele. Não seria mais correto pegar de lá?

File: designsystem/src/main/kotlin/com/natura/android/badge/BadgeDrawable.kt [hunk 3]
```diff
@@ -165,28 +163,45 @@ class BadgeDrawable(
 
     private fun getBackgroundColorByAttr(): Int {
```
Comment (yasminnvaz) [inline designsystem/src/main/kotlin/com/natura/android/badge/BadgeDrawable.kt:164]: Por que temos essa função repetida em dois lugares?

File: designsystem/src/main/kotlin/com/natura/android/badge/Badge.kt (hunk 1 above)
Comment (gsleonel) [reply designsystem/src/main/kotlin/com/natura/android/badge/Badge.kt:45]: Porque essa opção só é possível através de um atributo. Essa alternância só pode ser feita internamente.

File: designsystem/src/main/kotlin/com/natura/android/badge/BadgeDrawable.kt (hunk 3 above)
Comment (gsleonel) [reply designsystem/src/main/kotlin/com/natura/android/badge/BadgeDrawable.kt:164]: Por conta que o efeito de animação não pode ser feito na classe Drawable. Alterei para um nome mais nítido.

**Pull request merged**